lib/probe_dock_ruby/annotation.rb

```diff
@@ -0,0 +1,84 @@
+module ProbeDockProbe
+  class Annotation
+    ANNOTATION_REGEXP = /@probedock\(([^\(\)]*)\)/
+
+    attr_reader :key, :category, :tags, :tickets, :active
+
+    def initialize(str)
+      parse(str)
+    end
+
+    def merge(annotation)
+      @key = annotation.key if annotation.key
+      @category = annotation.category if annotation.category
+      @active = annotation.active unless annotation.active.nil?
+      @tags = (@tags + annotation.tags).compact.collect(&:to_s).uniq
+      @tickets = (@tickets + annotation.tickets).compact.collect(&:to_s).uniq
+      self
+    end
+
+    def self.strip_annotations(test_name)
+      test_name.gsub(ANNOTATION_REGEXP, '')
```

Comment:
The [node implementation](https://github.com/probedock/probedock-node/blob/2b3fb606b5c221a45116f386b4fb6d1a2b309d51/lib/testRun.js#L79) also strips trailing spaces. Adding a `.strip` would do the trick.